MuxStatsGoogleIMAPluginSPMExampleIOS/MuxStatsGoogleIMAPluginSPMExampleIOS/PlayerContainerViewController.swift

```diff
@@ -74,28 +87,14 @@ class PlayerContainerViewController: UIViewController {
         }
 
         // MARK: Setup Mux Data IMA Plugin
-
         imaListener = MuxImaListener(
-            playerBinding: playerBinding
-        )
-
-        NotificationCenter.default.addObserver(
-            self,
-            selector: #selector(Self.handleContentDidFinishPlaying(_:)),
-            name: NSNotification.Name.AVPlayerItemDidPlayToEndTime,
-            object: contentPlayer.currentItem)
-
-        // MARK: Setup Google IMA Ads
-
-        contentPlayhead = IMAAVPlayerContentPlayhead(
-            avPlayer: contentPlayer
+            playerBinding: playerBinding,
+            monitoringAdsLoader: adsLoader
         )
-
-        // TODO: viewWillAppear?
-        showContentPlayer()
-
-        adsLoader = IMAAdsLoader(settings: IMASettings())
-        adsLoader.delegate = self
+    }
+    
+    override func viewWillAppear() {
```

Comment:
Getting build errors locally: `Method does not override any method from its superclass`. This should be:

```suggestion
    override func viewWillAppear() {
        super. viewWillAppear(animated)
```